lib/reporters/junit/index.js

```diff
@@ -20,12 +20,15 @@ JunitReporter = function (newman, reporterOptions) {
             root,
             testSuitesExecutionTime = 0,
             executionTime = 0,
-            timestamp;
+            timestamp,
+            classname = _.camelCase(collection.name).replace(/\W/g, '');
 
         if (!report) {
             return;
         }
 
+        classname = classname[0].toUpperCase() + classname.slice(1);
```

Comment:
Since we're already bailing out in the `if` block above, this statement can be re-written as:

```suggestion
        classname = _.upperFirst(_.camelCase(classname));
```

Reply:
Thank you. Rewritten. I've left replacement of non-word characters so because implementation of `camelCase` in lodash won't filter some characters that can't be part of java class name.